src/js/auto-advance.js

```diff
@@ -54,7 +63,19 @@ const setup = (player, delay) => {
   player.one('ended', player.playlist.autoadvance_.trigger);
 };
 
+/**
+ * Used to change the reset function in this module at runtime
+ * This should only be used in tests.
+ *
+ * @param {Function} fn
+ *        The function to se the reset to
+ */
+const setReset_ = (fn) => {
```

Comment:
We'd want to have this not be available in production.
Another option is (for now) to switch the test to use proxyquireify or something.